lib/charms/nrpe_exporter/v0/nrpe_exporter.py

```diff
@@ -0,0 +1,342 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
Should this file be renamed to `nrpe_external_master`?

Reply:
It listens on numerous relations ,which can be specified as a `dict`. Primarily, the jobs actually appear on `nagios`, `local-monitors`, and `monitors`, so the library name here is intended to be reasonably generic. It's also unlikely to ever get published